backend/api/v1/setting_service.go

```diff
@@ -166,6 +166,13 @@ func (s *SettingService) SetSetting(ctx context.Context, request *v1pb.SetSettin
 			}
 			payload.GitopsWebhookUrl = gitopsWebhookURL
 		}
+		if payload.ExtraDetails != "" {
+			extraDetails, err := common.NormalizeExternalURL(payload.ExtraDetails)
```

Comment:
This method is for our own external-url setting,  not intended to be used to validate normal url.